app/src/js/components/ReconciliationReports/reconciliation-report.js

```diff
@@ -246,6 +246,7 @@ class ReconciliationReport extends React.Component {
             data={cardConfig[activeIdx].data}
             tableColumns={cardConfig[activeIdx].columns}
             shouldUsePagination={true}
+            initialHiddenColumns={['']}
```

Comment:
Eventually, this should be an array of column ids, indicating the columns that we want to have hidden on load